modules/virt-hot-plugging-memory.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-edit-vms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-hot-plugging-memory_{context}"]
+
+= Hot plugging memory on a virtual machine
+
+You can add or remove the amount of memory allocated to a virtual machine (VM) without having to restart the VM by using the {product-title} web console or editing the YAML file.

```

Comment:
I'm not sure it's obvious to users what "the" YAML file is in this context, or whether that option is even documented in this procedure. If it's not feasible to resolve this before GA then that's fine, but could you add the name or an adjective for the YAML file you're referring to?